tests/Unit/inc/Engine/License/API/PricingClient/getPricingData.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit\inc\Engine\License\API\PricingClient;
+
+use Brain\Monkey\Functions;
+use Mockery;
+use WP_Rocket\Engine\License\API\PricingClient;
+use WP_Rocket\Tests\Unit\TestCase;
+
+/**
+ * @covers \WP_Rocket\Engine\License\API\PricingClient::get_pricing_data
+ *
+ * @group License
+ */
+class GetPricingData extends TestCase {
+	/**
+	 * @dataProvider configTestData
+	 */
+	public function testShouldReturnExpected( $config, $expected ) {
+		$client = new PricingClient();
+
+		if ( true === $config['transient'] ) {
+			Functions\expect( 'get_transient' )
+				->once()
+				->with( 'wp_rocket_pricing' )
+				->andReturn( $expected );
+		} else {
+			Functions\expect( 'get_transient' )
+				->once()
+				->with( 'wp_rocket_pricing' )
+				->andReturn( false );
+		}
```

Comment:
Refactor to simplify:

```suggestion
		Functions\expect( 'get_transient' )
			->once()
			->with( 'wp_rocket_pricing' )
			->andReturn( $true === $config['transient'] ? $expected : false );
```